src/metax_api/services/common_service.py

```diff
@@ -469,3 +469,24 @@ def set_if_modified_since_filter(cls, request, filter_obj):
                 filter_obj['q_filters'].append(flter)
             else:
                 filter_obj['q_filters'] = [flter]
+
+    @staticmethod
+    def identifiers_to_ids(identifiers, params=None):
+        """
```

Comment:
Method could have Type check for list:
`def identifiers_to_ids(identifiers:List[Any], params=None):`